src/package/Dependencies.js

```diff
@@ -85,7 +85,8 @@ class Dependencies extends AbstractService {
       console.log(chalk.white(await this.run(this.plugin.settings.customInstallationCommand)));
     } else {
       const commands = this.plugin.runtimes.getCommands();
```

Comment:
`commands` is not used anymore my linter is not liking it

Reply:
@Shereef I though it worth logging the final command like:

version: `serverless-layers@2.5.2-beta.3`

<img width="469" alt="Screen Shot 2022-02-10 at 9 30 35 pm" src="https://user-images.githubusercontent.com/298845/153388777-f91c3521-2152-4148-a920-8e4d7ec04a94.png">
<img width="460" alt="Screen Shot 2022-02-10 at 9 29 36 pm" src="https://user-images.githubusercontent.com/298845/153388800-b080bf1f-78e9-489f-8904-9d99a5644069.png">

Reply:
oh no I was in your project debugging to understand why I had the failure
then i deduced it was `install` not in the command after `npm`

and since I am reviewing this PR I thought I would write it for you to optionally fix

Reply:
Perfect! thanks for logging the command!